docs/workflow/blocks/form.rst

```diff
@@ -44,21 +44,83 @@ not dependent on the form, then it is best to place them before the form in the
 
 Config init
 -----------
-If can often be beneficial to initialiase the form data by creating an mapping of the default data. 
+It can often be beneficial to initialiase the form data by creating a mapping of the default data. 
```

Comment:
I noticed 'initialiase' is not spelt correctly and that it wasn't clear English then I came up with this:
```suggestion
To fill out existing information easily, a form can be loaded with pre-existing fields, using a 'mapping' - a rule that determines how to generate the default data.
```

Reply:
Mhm honestly both are not clear to me. If I would be new to Kendraio project I would have no idea what we are talking about :D 
I suggest to add an example that is more helpful then 
```

  {
    "confirm": false
  }

```
I will commit the changes and then we can discuss on the new text/example